rule/utils.go

```diff
@@ -112,6 +113,14 @@ func checkNumberOfArguments(expected int, args lint.Arguments, ruleName string)
 	return nil
 }
 
+// normalizeRuleOption returns an option name from the argument. It is lowercased and without hyphens.
+// This function is used in the Rule.Configure method.
+//
+// Example: normalizeRuleOption("allowTypesBefore"), normalizeRuleOption("allow-types-before") -> "allowtypesbefore"
+func normalizeRuleOption(arg string) string {
+	return strings.ToLower(strings.ReplaceAll(arg, "-", ""))
+}
+
```

Comment:
I wonder if a function like this couldn't help for clarity purpose


```suggestion
func normalizeRuleOption(arg string) string {
	return strings.ToLower(strings.ReplaceAll(arg, "-", ""))
}

func isRuleOption(a, b string) {
	return normalizeRuleOption(a) == normalizeRuleOption(b)
}
```